types/src/fee_statement.rs

```diff
@@ -0,0 +1,168 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::bail;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+pub enum FeeStatement {
+    V0 {
+        /// maps to `V0::gas_used`
+        total_charge_gas_units: u64,
+    },
```

Comment:
Because this is entirely internal, do we really need v0? or does it even need to be an enum instead of a struct?